src/components/Home.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Home: React.FC = () => {
```

Comment:
```suggestion
export const HomePage: React.FC = () => {
```
Also change file name